README.md

```
@@ -24,7 +24,7 @@ A powerful, extensible, and runtime command terminal for Unity.
 
 -   **🖥️ ランタイム実行:** ゲーム実行中にリアルタイムでコマンドを呼び出し。
 -   **✍️ 柔軟なコマンド登録:**
-    -   <b>自動登録</b>: C#のstaticメソッドに[Register]属性を付けるだけで、コマンドとして自動認識。
+    -   <b>自動登録</b>: C#のstaticメソッドに[TerminalCommand]属性を付けるだけで、コマンドとして自動認識。
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)